basket/petition/views.py

```diff
@@ -59,7 +40,9 @@ def sign_petition(request):
         petition.save()
```

Comment:
should we also catch a potential DB error here and specifically return an error, or just let it 500?